src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java

```diff
@@ -57,20 +57,85 @@
  * </li>
  * </ul>
  * <p>
- * To configure the check to report on the first instance in each file:
+ * To configure the check to report only the first instance in each file:
  * </p>
  * <pre>
  * &lt;module name=&quot;FileTabCharacter&quot;/&gt;
  * </pre>
  * <p>
- * To configure the check to report on each line in each file:
+ * Example - Test.java:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   int a;            // violation : indented using tab, no further checks
+ *
+ *   public void foo (int arg) {
```

Comment:
please add `// ok : indented using tab, as only first occurrence in file reported`

please add to this case comments like:
`// OK, indented using spaces`

Please make all 3 Test.java files to be same by comment presence (diff only in content of comments).

I know that you have `, no further checks` , but I found it not easy to catch.

Reply:
Ok, done, but some comments are too close to the code to keep line length under 100.